earth_enterprise/src/fusion/autoingest/MiscConfigStorage.idl

```diff
@@ -28,8 +28,8 @@ class MiscConfigStorage {
   uint        VersionCacheSize = uint(128000);
 
   // The maximum amount of memory allocated for each cache in bytes.
-  uint64      MaxAssetCacheMemorySize = uint64(1073741824);
-  uint64      MaxVersionCacheMemorySize = uint64(1073741824);
+  uint64      MaxAssetCacheMemorySize = uint64(10737418240);
```

Comment:
We may need to think about what the default memory size should be. I'm not sure it's reasonable to assume that everyone has 20 GB of RAM. Our [system requirements](http://www.opengee.org/geedocs/5.3.2/answer/installGEE/sysReqGEEServer.html) state users need 4 GB of RAM and recommend at least 8 for Fusion, so maybe we should default to using 3 or 7 GB combined.

Along the same lines, assets tend to be much smaller than asset versions, so it might be wise to have an unequal distribution of memory between them. I could see giving 4x or 8x as much memory to the asset versions as to the assets.

I'm happy to hear other thoughts on this as well. There's not a clear right answer.